Makefile

```diff
@@ -217,6 +217,14 @@ launch-dev: check-kubectl check-kubectl-connection install
 	until timeout 1s ./etc/kube/check_ready.sh app=pachd; do sleep 1; done
 	@echo "pachd launch took $$(($$(date +%s) - $(STARTTIME))) seconds"
 
+launch-enterprise: check-kubectl check-kubectl-connection install
```

Comment:
I think this change is also in the other P.R.  Double check this merges cleanly before it goes in.